Start/readme2.md

```diff
@@ -0,0 +1,55 @@
+## 4.3.3 &ensp; Users Perspective: The Front View
+The various aspects an users expects from the system and which is my one of the main ideal outcomes of the project can be viewed from the following figure:
+
+<p align=center>
+<img src="Start\user perspective.png" height=390px width=390px>
+</p>
```

Comment:
@smriti1313 Again html tags. Actually we want this file to be written in a complete markdown format without any inclusion of html tags and attributes. :worried: